data/tokens.json

```diff
@@ -1227,56 +1227,56 @@
         "type": "color"
       }
     },
-    "gtmds_action-item-margin-right": {
+    "gtmds_action-item_margin-right": {
       "value": "{gtmds_size-sm}",
       "type": "spacing"
     },
-    "gtmds_action-item-margin-bottom": {
+    "gtmds_action-item_margin-bottom": {
       "value": "{gtmds_size-sm}",
       "type": "spacing"
     },
     "gtmds_action-item_padding": {
       "value": "{gtmds_size-md}",
       "type": "spacing"
     },
-    "gtmds-actionItem-sizeWidth": {
+    "gtmds_action-item_size-width": {
       "value": "auto",
       "type": "sizing"
     },
-    "gtmds-actionItem-sizeHeight-default": {
+    "gtmds_action-item_size-height": {
       "value": "{gtmds_size-xl}+{gtmds_size-md}+{gtmds_size-sm}",
       "type": "sizing"
     },
-    "gtmds-actionItem-sizeHeight-description": {
+    "gtmds_action-item_size-height-description": {
```

Comment:
```suggestion
    "gtmds_action-item-description_size-height": {
```

Reply:
@Ddungyov if you agree with the recommendation you can resolve it directly fro here by clicking "Commit suggestion" above